examples/demo/canvas/canvas.py

```diff
@@ -25,7 +25,7 @@
 
 # Chaco imports
 from chaco.api import AbstractOverlay, ArrayPlotData, \
-        Plot, jet, ScatterPlot, LinePlot, LinearMapper
+        Plot, viridis, ScatterPlot, LinePlot, LinearMapper
```

Comment:
again, looks like we're not using the `viridis` color map in this plot but im not sure what color map is being used.

Reply:
Yeah I don't think a color map is used anywhere, they're all just line plots with the color explicitly set to blue.